src/utils/chartJs/CustomTooltip.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  tooltip: {
+    padding: '6px',
+    background: 'rgba(0, 0, 0, .8)',
+    color: 'white',
+    borderRadius: '4px',
+    pointerEvents: 'none',
+  },
+  tooltipKey: {
+    display: 'inline-block',
+    width: '10px',
+    height: '10px',
+    marginRight: '10px',
+  },
+};
+
+function CustomTooltip({ classes, tooltipModel, series }) {
+  if (tooltipModel.opacity === 0) return null;
+
+  const currPoint = tooltipModel.dataPoints[0];
+  const labelColors = tooltipModel.labelColors[0];
+  const { index } = currPoint;
+  const currSeries = series[currPoint.datasetIndex];
+  const higherIsBetter = !currSeries.meta.lower_is_better;
+  const higherOrLower = higherIsBetter ? 'higher is better' : 'lower is better';
+  const curr = currSeries.data[index];
+  const hgURL = `https://hg.mozilla.org/mozilla-central/pushloghtml?changeset=${
+    curr.revision
+  }`;
+  const jobURL = `https://treeherder.mozilla.org/#/jobs?repo=mozilla-central&revision=${
+    curr.revision
+  }&selectedJob=${curr.job_id}&group_state=expanded`;
```

Comment:
use this

```javascript
const jobURL = `https://treeherder.mozilla.org/#/jobs?${
  Object2URL({
    repo: "mozilla-central",
    revision: curr.revision,
    selectedJob: curr.job_id,
    group_state: "expanded"
})}`
```

generally, we would like to avoid string concatenation as much as possible